frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -178,6 +178,27 @@ action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=Slide
 action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected branch
 
 
+# Graph Table
+
+action.GitMachete.DiscoverAction.text=Discover Branch Layout
+action.GitMachete.DiscoverAction.description=Discover Branch Tree in Git Machete Plugin
```

Comment:
`Discover Branch Layout Tree`? Referencing "plugin" here looks weird :thinking: 
Or maybe: 
`Git Machete: Discover Branch Layout Tree`
since it's gonna be called via `Find Actions`

Reply:
ok, the latter sounds good

Reply:
or maybe even former is the best